sql/moz-fx-data-shared-prod/fenix_derived/meta_attribution_country_counts_v1/query.sql

```diff
@@ -0,0 +1,25 @@
+-- Aggregate meta attribution counts by country and channel.
+SELECT
+  DATE(submission_timestamp) AS date_attributed,
+  CASE
+    WHEN (metrics.string.meta_attribution_app = '382348575493443')
+      THEN 'Firefox Focus for Android'
+    WHEN (metrics.string.meta_attribution_app = '979253712091854')
+      THEN 'Firefox Browser for Android'
+    WHEN (metrics.string.meta_attribution_app = '697946762208244')
+      THEN 'Firefox Nightly for Android'
+    ELSE NULL
+  END AS meta_attribution_app,
+  normalized_channel,
+  metadata.geo.country AS country,
+  COUNT(*) AS client_count
```

Comment:
This is ping count, not client count
```suggestion
  COUNT(*) AS ping_count,
  COUNT(DISTINCT client_id) AS client_count,
```

Reply:
same for `init.sql`